jwst/extract_1d/extract.py

```diff
@@ -2110,7 +2114,10 @@ def extract(self, data, wl_array, verbose):
             The background count rate that was subtracted from the total
             source count rate to get `net`.
 
-        dq : ndarray, 1-D, int32
+        npixels : ndarray, 1-D, float32
```

Comment:
data type should be listed as float64, right?

Reply:
Yes, thank you!  I missed these two comments; I'll fix them.